README.md

```diff
@@ -205,6 +211,19 @@ The Diffie-Hellman group should have at least 1024 bit and can be created with t
 ```
 # openssl dhparam -out /etc/exim4/dhparam.pem 2048
 ```
+In the transport section, you need to modify the user and group to the exim-user `system_aliases`-transport (around line 800):
+```
+system_aliases:
+  driver = redirect
+  allow_fail
+  allow_defer
+  data = ${lookup{$local_part}lsearch{/etc/aliases}}
+  user = <exim-user>
+  group = <exim-group>
+  file_transport = address_file
+  pipe_transport = address_pipe
```

Comment:
1. I think this is the router section, not transport. At least in Debian this is `conf.d/router/400_exim4-config_system_aliases`
2. Are these last four lines actually necessary? Again, in Debian they are all placed under `.ifdef`'s, hinting that they might be optional after all. If they are, we can just skip them, and then drop this section from the manual, right?